README.md

```diff
@@ -113,51 +167,94 @@ You need to set up these environment variables:
     -   Used when importing or exporting coverage reports
 -   `COVERAGE_VERBOSE=1` to see logs (optional)
 
-Another way to enable coverage is to use the Meteor --settings file.
-
 Using `spacejam --coverage` you do not have to set up global environment variable.
 
-## My files are missing from my app coverage report
-
-If you have **internal packages** inside your app and you want to get their **server side** coverage. Open the file `.meteor/packages` and move the line `lmieulet:meteor-coverage` to be above these packages.
-
 ## Config file
 
 If you have packages used by your project (ex: aldeed:simple-schema) or libraries on your client side (ex: OpenLayers, Jquery), you can hide the coverage of these files from reports. You can specify which files will not be covered in a `.coverage.json` file inside the `COVERAGE_APP_FOLDER` folder.
 
 If you do not have this file, this package will use the default one (`conf/default-coverage.json`). If you do not define a key in the `.coverage.json` file, the default one will be used.
 
-Copy the `conf/default-coverage.json`, rename it into `.coverage.json`, remove keys that you don't want to overwrite and pimp it.
-
+Exemple:  
 ```json{
- "include": [],
- "exclude": {
-   "client": [
-     "You may want to exclude here modules from client side instrumentation",
-     "to get the lightest version of your browser app",
-     "**/underscore.js",
-     "**/meteor.js",
-     "...",
-     "And even dependencies / internal packages:",
-     "/lmieulet_meteor-coverage.js",
-     "==> if you cloned a package into your package directory",
-     "it will ignores the minified version of the package from client instrumentation"
-   ],
-   "server": [
-    "same thing than client. "
-   ],
-   "general": [
-     "here you can ignore anything from your project coverage report",
-     "**/tests/*.js"
-   ]
- },
- "output": "./.coverage"
+  "include": [
+    "**/packages/author_packageName.js"
+  ],
+  "exclude": {
+    "general": [
+        "We will keep this one empty to let you easily exclude anything using the glob pattern"
+    ],
+    "server": [
+      "**/.?*/**",
+      "**/packages/!(local-test_?*.js)",
+      "**/+([^:]):+([^:])/**",
+      "**/@(test|tests|spec|specs)/**",
+      "**/@(test.?*|?*.test.?*|?*.tests.?*)",
+      "**/@(spec.?*|?*.spec.?*|?*.specs.?*)",
+      "**/@(app-test.?*|?*.app-test.?*|?*.app-tests.?*)",
+      "**/@(app-spec.?*|?*.app-spec.?*|?*.app-specs.?*)"
+    ],
+    "client": [
+      "**/client/stylesheets/**",
+      "**/.npm/package/node_modules/**",
```

Comment:
The glob pattern `**/.npm/package/node_modules/**` is already catched by `**/.?*/**`, which excludes **any** hidden folder, but it does not cover `node_modules` folder in a meteor app (in a meteor app, the used NPM modules are installed at `node_modules`, not `.npm/package/node_modules`), and that's why I set it up as `**/node_modules/**`. I'm developing a meteor app with `angular2-meteor`, and all the NPM packages are saved to `node_modules/**`, so the current glob pattern presumably will not catch it (not tested yet, I'm focused on meteor package testing right now).